src/main/java/com/arpnetworking/tsdcore/sinks/WavefrontSink.java

```diff
@@ -66,9 +67,25 @@ public void recordAggregateData(final PeriodicData periodicData) {
             // Build the list of annotations based on cluster and white listed dimensions
             final Map<String, String> annotations = Maps.newHashMap();
             annotations.put("cluster", fqdsn.getCluster());
-            periodicData.getDimensions().entrySet().stream()
-                    .filter(entry -> _whiteListedDimensions.contains(entry.getKey()))
-                    .forEach(entry -> annotations.put(entry.getKey(), entry.getValue()));
+            if (_whiteListedDimensions.size() == 0) {
```

Comment:
You need to make the list optional instead of default to empty because you need to tell the difference between someone specifying an empty list versus no list specified.